File: Arrays/check_duplicate.js
var containsDuplicate = function(nums) {
    const myMap = new Map();
    for(var i in nums){
        if(myMap.has(nums[i])){
           return true;
        }else{
             myMap.set(nums[i], i);
        }
    }
    return false;
    
};

File: Algor_Questions/Arrays/check_duplicate.py
# Link: https://leetcode.com/problems/contains-duplicate/

class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        set_len = len(set(nums))
        list_len = len(nums)
        if set_len != list_len: return True
        else: return False
